frontend/src/lib/k8s/service.ts

```diff
@@ -51,7 +51,7 @@ class Service extends makeKubeObject<KubeService>('service') {
     return _.uniq(
       (
         this.status?.loadBalancer?.ingress?.map(
-          (ingress: KubeLoadBalancerIngress) => ingress.hostname || ingress.ip
+          (ingress: KubeLoadBalancerIngress) => ingress?.hostname || ingress?.ip
```

Comment:
Hi @lijianzhi01 . I would assume `ingress` to be an array and never have `null` in.
Are we preprocessing that array somehow and leading to those null refs?

Reply:
Yes. Let me see if I can find any evidence for null value of ingress.